### PR TITLE
Update look for wms and geojson views

### LINKED ISSUE #28: Update geojson and wms layer views to match multimap and map index views.

### PR DESCRIPTION
**Pull request by grimm**
Fixes #28

### CODE REVIEW

**Comment by grimm** [comment]
Ready for review.

File: app/controllers/geojson_layers_controller.rb [hunk 1]
```diff
@@ -2,7 +2,7 @@ class GeojsonLayersController < ApplicationController
   before_action :set_map, only: [:remove, :add]
   before_action :set_geojson_layer, only: [:show, :edit, :update, :destroy, :remove, :add]
 
-  #layout 'manager'
+  layout 'manager'
```
Comment (teknofire) [inline app/controllers/geojson_layers_controller.rb]: Don't need this with the latest master code, instead have this controller inherit the `ManagerController`

File: app/views/geojson_layers/_option_fields.html.haml [hunk 2]
```diff
@@ -11,3 +11,5 @@
     %label.checkbox
       = opts.check_box :auto_zoom, { }, 'yes', 'no'
       Auto zoom
+    .form-group
+      = link_to 'Delete This Layer', @geojson_layer, method: :delete, data: { confirm: 'Are you sure?' }, class: 'btn btn-danger btn-block'
```
Comment (teknofire) [inline app/views/geojson_layers/_option_fields.html.haml]: Probably should just have it say 'Delete' with the trash icon

File: app/views/geojson_layers/index.html.haml [hunk 3]
```diff
@@ -1,24 +1,23 @@
-%h1 Listing geojson_layers
-.row
-  .col-lg-2.col-md-3.col-sm-3
-    = link_to 'New Layer', new_geojson_layer_path, class: 'btn btn-success btn-block'
-  .col-lg-10.col-md-9.col-sm-9
-    %table.table.table-striped.table-condensed
-      %thead
+- content_for :side_menu do
+  = link_to 'New Layer', new_geojson_layer_path, class: 'btn btn-success btn-block'
```
Comment (teknofire) [inline app/views/geojson_layers/index.html.haml]: have the link say 'New GeoJSON Layer'

File: app/views/wms_layers/_option_fields.html.haml [hunk 4]
```diff
@@ -22,3 +22,5 @@
     .form-group
       = opts.label :slider_title
       = opts.text_field :slider_title, class: 'form-control'
+    .form-group
+      = link_to 'Delete This Layer', @wms_layer, method: :delete, data: { confirm: 'Are you sure?' }, class: 'btn btn-danger btn-block'
```
Comment (teknofire) [inline app/views/wms_layers/_option_fields.html.haml]: Again, just say 'Delete' with trashcan icon

File: app/views/wms_layers/index.html.haml [hunk 5]
```diff
@@ -1,22 +1,21 @@
-%h1 Listing wms layers
-.row
-  .col-lg-2.col-md-3.col-sm-3
-    = link_to 'New layer', new_wms_layer_path, class: 'btn btn-success btn-block'
-  .col-lg-10.col-md-9.col-sm-9
-    %table.table.table-condensed
-      %thead
+- content_for :side_menu do
+  = link_to 'New layer', new_wms_layer_path, class: 'btn btn-success btn-block'
```
Comment (teknofire) [inline app/views/wms_layers/index.html.haml]: should say `New WMS Layer`

**Comment by grimm** [comment]
Ready for review.

**Pull request merged**